### PR TITLE
add for #238

### PR DESCRIPTION
**Pull request by pjaudiomv**

### CODE REVIEW

**Comment by radius314** [comment]
holding for yap 3.1

**Comment by pjaudiomv** [comment]
I have tested this thoroughly in all use cases. What's your reasoning in waiting?

**Comment by radius314** [comment]
Ok, can you update release notes?

**Comment by pjaudiomv** [comment]
Sure thing

File: RELEASENOTES.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Release Notes
 ### 3.0.0 (unreleased)
+* The sms_summary_page not and sms_ask option can now be used together.
```
Comment (radius314) [inline RELEASENOTES.md]: The description here doesn't seem to make sense.  Can you move bug fixes to the bottom of the list?

Comment (pjaudiomv) [reply RELEASENOTES.md]: Hmm weird yes that is confusing will do

Comment (radius314) [reply RELEASENOTES.md]: I forgot to include the ticket number in each enhancement / bug fix.  You should follow the pattern established in prior release notes.

**Comment by radius314** [comment]
Thanks for adding all the ticket numbers, much appreciated!

**Pull request merged**